scripts.py

```diff
@@ -930,7 +940,7 @@ def _run_migration(self, migration_path, timestamp):
                 sql = clause.read()
                 self._db.execute(sql)
         if migration_path.endswith('.py'):
-            module_name = migration_filename
+            module_name = migration_filename[:-3]
```

Comment:
This was causing an error for every global "import" in the migration file. Gross.